department-of-veterans-affairs/caseflow
# Removing some VACOLS hearing day functionality

### Description
I was making some unrelated code changes when I realized it was possible we didn't have AMA support for creating CO hearings. Then I realized that our code for slotting hearings is way too confusing and no longer has to differentiate between creating video and co records. While fixing that, I realized that hearing coordinators shouldn't be slotting veterans for hearings before 4/1 anymore, so I removed that functionality as well.

### Testing Plan
1. Confirm we can schedule a CO AMA hearing
1. Confirm we can schedule a CO legacy hearing
1. Confirm we can schedule a video AMA hearing
1. Confirm we can schedule a video legacy hearing
